samples/secure-fraud-detection/src/main/java/io/quarkiverse/langchain4j/sample/ProvidersConfigSource.java

```diff
@@ -0,0 +1,29 @@
+package io.quarkiverse.langchain4j.sample;
+
+import java.util.Map;
+import java.util.Set;
+
+import org.eclipse.microprofile.config.spi.ConfigSource;
+
+public class ProvidersConfigSource implements ConfigSource {
```

Comment:
Can you explain what this does and why it's needed?

Reply:
Sure, this is more to support the message the whole setup is about security (google client id, secret and open ai key are not in the open application.properties), I can drop it if you'd like

Reply:
We don't need to drop it necessarily, but if we do keep it I'd like to have some Javadoc on what this does and why it's necessary

Reply:
Sure

Reply:
Hi @geoand I'll drop it in favor of proposing to use env properties as shown in the cli translate demo, the fewer resources the better :-), and it will be the same recommendation across at least 2 demos